### PR TITLE
Add migrations to docker-compose up

### PR DESCRIPTION
**Pull request by vibby**
Hello, here is an approach for migrations in ```docker-compose up```. But, it’s not fully working, especially on the first ```up```.

Also, I speed up composer with cache sharing.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -50,11 +48,26 @@ services:
...
+    command: install --ignore-platform-reqs
+    volumes:
+      - ./application:/app
+      - ./var/cache/composer:/tmp
+
+  ac-migration:
+    restart: 'no'
+    build:
+      context: ./docker/php-fpm
+    command: ["application/vendor/bin/phinx", "migrate"]
```
Comment (davrv93) [inline docker-compose.yml]: This are the lines about migration on start up right? If so, we can wait for them to work :)

File: docker-compose.yml [hunk 2]
```diff
@@ -50,11 +48,26 @@ services:
...
+    volumes:
+      - .:/var/www/adventistcommons:ro
+    working_dir: /var/www/adventistcommons
+    links:
+      - ac-db
+    depends_on:
+      - ac-db
+      - ac-composer
```
Comment (davrv93) [inline docker-compose.yml]: Thanks Vincent for your contribution. I think we can work a little bit more on the "Add migrations to docker-compose up" functionality for make it work always =)  :+1:  God bless you!

Comment (vibby) [reply docker-compose.yml]: I think you’re right :)
I also tried by using this image : https://hub.docker.com/r/vegbrasil/phinx/dockerfile but I finally think it is not a good idea, this phinx version is blocked at 8.x in it.

**Comment by davrv93** [comment]
I'm closing this PR as we coordinate with @vibby  it would be better to wait for the full feature working.

**Comment by vibby** [comment]
+1

**Pull request closed without merging**